lib/l10n/intl_en.arb

```diff
@@ -190,16 +192,33 @@
-  "discovery_navbar_schedule_page_title": "Schedule",
-  "discovery_navbar_schedule_page_details": "The Schedule page allows you to view the events of your school calendar.",
-  "discovery_navbar_student_page_title": "Student",
-  "discovery_navbar_student_page_details": "The Student page displays your grades and profile information.",
-  "discovery_navbar_ets_page_title": "ÉTS",
-  "discovery_navbar_ets_page_details": "The ÉTS page allows you to quickly consult all of the important ÉTS links, such as Moodle, MonÉTS and security measures.",  
-  "discovery_navbar_more_page_title": "More",
-  "discovery_navbar_more_page_details": "The More page gives you access to additional options of the application.",
+
+  "discovery_navbar_dashboard_title": "Dashboard",
+  "discovery_navbar_dashboard_details": "The Dashboard page groups together the most frequently consulted information, in the form of cards.\n\nIf you keep your finger on the card, you can move it up and down to rearrange the cards.\n\nTo delete a card, swipe it to the left or right.",
+  "discovery_page_dashboard_restore": "To restore the changes, click on the icon at the top right.",
+  
+  "discovery_navbar_schedule_title": "Schedule",
+  "discovery_navbar_schedule_details": "The Schedule page allows you to view the events of your school calendar.",
+  "discovery_page_schedule_settings": "In the settings, you can change the calendar format and the first day of the week displayed.",
+  
+  "discovery_navbar_student_title": "Student",
+  "discovery_navbar_student_details": "The Student page displays your grades and profile information.",
+  
+  "discovery_page_student_grades_session": "Your courses are divided by session, with the most recent at the top. At any moment, you can drag the list to refresh it.",
+  "discovery_page_student_grades_grade_button": "Here you see your current grades. By tapping on it, you will be able to see the details of all the evaluations obtained in the course.",
+  
+  "discovery_page_student_profile": "Here you will find all the information related to your student account, from the balance to your active programs through your permanent code.",
+  
+  "discovery_navbar_ets_title": "ÉTS",
+  "discovery_navbar_ets_details": "The ÉTS page allows you to quickly consult all of the important ÉTS links, such as Moodle, MonÉTS and security measures.",  
+  
+  "discovery_navbar_more_title": "More",
+  "discovery_navbar_more_details": "The More page gives you access to additional options of the application.",
+  "discovery_page_more_report_bug": "The app is not perfect and, we are always open to suggestions! If you see a problem or want us to be part of an idea, you can use this feature.",
+  "discovery_page_more_contributors": "Here you will find the list of all the students who worked on the project from a code point of view.",
```

Comment:
I find that the lasts few words worded a bit weird, maybe if we say "Here you will find the list of all the students who contributed and developed this application." would it be better? any other idea?

Reply:
Bien vue, je devais être fatiguée quand j'ai écris ça xD